modules/running-insights-operator-gather.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+
+:_content-type: PROCEDURE
+[id="running-insights-operator-gather_{context}"]
+= Running an Insights Operator gather operation
+
+You can run Insights Operator data gather operations on-demand. The following procedure describes how to run the default list of gather operations excluding the `workloads` operation. You can customize the on-demand gather function to exclude any gather operations you choose. If you have previously disabled Insights Operator gather operations in your cluster, this procedure will override those parameters.  
+
+:FeatureName: The `DataGather` custom resource 
+include::snippets/technology-preview.adoc[] 
+
+[NOTE]
+====
+If you enable Technology Preview in your cluster, Insights Operator runs gather operations in individual pods. This is part of the Technology Preview feature set for the Insights Operator and supports the new data gathering features.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. To run the gather operation from the web console:
+.. Navigate to *Administration* -> *CustomResourceDefinitions*.
+.. On the *CustomResourceDefinitions* page, use the *Search by name* field to find the *DataGather* resource definition and click it.
+.. On the *CustomResourceDefinition details* page, click the *Instances* tab.
+.. Click *Create DataGather*.
+.. To create a new `DataGather` operation, edit the configuration file:
++
+[source,yaml]
+----
+apiVersion: insights.openshift.io/v1alpha1
+kind: DataGather
+metadata:
+  name: <your_data_gather> <1>
+spec:
+ gatherers: <2>
+   - name: workloads 
+     state: Disabled
+----
++
+--
+<1> Replace the `<your_data_gather>` with a unique name for your gather operation.
+<2> Enter individual gather operations to disable the under the `gatherers` parameter. This example disables the `workloads` data gather operation and will run the remainder of the default operations. To run the complete list of default gather operations, leave the `spec` parameter empty. You can find the complete list of gather operations in the Insights Operator documentation. 
```

Comment:
I think there's a word missing here: "to disable the <something> under the `gatherers` parameter."